mocks.pkg/src/config/base.config.ts

```diff
@@ -107,6 +109,8 @@ export const baseArgsConfig: ServicesArgsConfig = {
     getProductionVaults: { args: [] },
     getVaults: { args: [VaultVersion.v1, bcrvBadgerAuthor] },
     hasRegistry: { args: [] },
+    keysCount: { args: [] },
+    getFilteredProductionVaults: { args: [VaultVersion.v1_5, VaultState.Open] },
```

Comment:
should we add a mock for non filtering / support for it? 
can be fllow up

Reply:
wym? `getProductionVaults: { args: [] },` - its bit above

Reply:
i'm blind